python/hail/methods/io.py

```diff
@@ -1,18 +1,114 @@
 from hail.typecheck import *
 from hail.utils.java import Env, handle_py4j, joption
-from hail.api2 import MatrixTable
+from hail.api2 import Table, MatrixTable
 from hail.history import *
 from hail.expr.types import *
 from .misc import require_biallelic
 
+@handle_py4j
+@typecheck(table=Table,
+           address=strlike,
+           keyspace=strlike,
+           table_name=strlike,
+           block_size=integral,
+           rate=integral)
+def export_cassandra(table, address, keyspace, table_name, block_size=100, rate=1000):
+    """Export to Cassandra.
+
+    Warning
+    -------
+    :meth:`export_cassandra` is EXPERIMENTAL.
```

Comment:
I think this should be `` :func:`.export_cassandra` ``. That's the proper semantic markup, and it looks like with `:meth:` sphinx doesn't find the reference, and doesn't add the link in the html.

See http://www.sphinx-doc.org/en/stable/domains.html#python-roles